docs/DeveloperGuide.adoc

```diff
@@ -800,18 +812,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new person (tutor or student) |expand user's addressbook for future tuition coordination
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a person |remove entries that the user no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |sort contacts based on the clients' location (North, South, East, West, Central |better match students and tutors that are staying near each other.
+
+|`* * *` |user |edit the information stored |keep the information up to date
+
+|`* * *` |user |tag tutors and students with multiple fields |do multi-layer searching to better match tutors and students
+
+|`* * *` |user |categorise contacts between students and tutors |reduce the number of tagging required and make searching more convenience
+
+|`* * *` |user |group clients based on the subject they wish to study/teach (Math, Eng, Phy, Chem...) |find and match client easily by the subjects registered
+
+|`* * *` |user |record the gender preference the students/tutors have (male/female) |find and match client easily by the gender preference registered
+
+|`* * *` |user |see and compare the price that students/tutors are willing to pay/accept|match them based on pricing
+
+|`* * *` |user |have an auto-save function |changes will be saved once user exit the application
```

Comment:
This does not seem to fit into a user story - this seems to be out of the control of the user (since it's automatic after all) and would be more suited as a NFR

Reply:
Will update in my commit